tests/src/harness/shells.rs

```diff
@@ -41,9 +41,12 @@ impl Shell {
             Shell::Bash => format!(
                 r#"#Autogenerated by pazi integ tests
 set -e
+set -u
 {3}
 export PS1="{2}" # sep so we know when our commands finished
 export PATH=$PATH:$(dirname "{0}")
+export __PAZI_LAST_PWD=''
+export PROMPT_COMMAND=''
```

Comment:
I don't like this.

I think we should use the more traditional solution of fixing our case/if statements to be `${VARIABLE:-}` to dfeault to `''` on read without a `-u` error.

If, for some reason, we do need to initialize these, it should be before the `preinit` bit so `preinit` can change them.

Reply:
Strongly agreed here!